src/IPDPProvingSchedule.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+/// @title IPDPProvingWindow
+/// @notice Interface for PDP Service SLA specifications
```

Comment:
Perhaps note that this is specific to the SimplePDPService implemented here, and other services could have different models (e.g. without proving periods).